sig/linglong/metadata.yml

```
@@ -0,0 +1,23 @@
+name: linglong
+blog:
+rss:
```

```suggestion
rss:
matrix: '#linglong:deepin.org'
```

另外目前默认会给每个新的小组提供一个现成的静态博客用来放小组的进度公示之类的博客内容（使用 deepin-community/sig-template 作为模板仓库，创建到 deepin-community/sig-linglong 之下），这个小组需要吗？需要的话可以考虑吧 rss 和 blog 也写上，这样到时候会自动获取相关的内容。

---

好的👌

---

诶 具体来说 地址写什么呢

---

可以参照文档组：

https://github.com/deepin-community/SIG/blob/22f010f1e5520673dc7860945cf8e06920d1c443/sig/deepin-doc-doc-go/metadata.yml#L3-L4

sig-deepin-doc-doc-go 替换为 sig-linglong